### PR TITLE
7831.main - Multistep Form

### PR DESCRIPTION
**Pull request by KuzyaWkk**
https://redmine.devbranch.work/issues/7831

Created an element multistep ui table that extends the table. 
Created a multistep form for ordering a product. 
Also created a configuration form that allows you to swap steps and disable certain steps.
Two different configuration forms have been created, one of which uses the multistep ui table element renderer. 
The other form provides similar functionality but with a crutch, without its own js.
Added js for the configuration form with multistep ui table using JQuery.

### CODE REVIEW

**Review by faust174 — APPROVED** [review]
LGTM

File: web/modules/custom/multistep_form/src/Form/MultistepForm.php [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+    $form['link'] = [
+      '#type' => 'link',
+      '#title' => $this->t('Configure your form here'),
+      '#url' => Url::fromRoute('multistep_form.multistep_form_settings_js'),
+      '#attributes' => ['class' => ['button__configure-form']],
+    ];
+
+    if ($max_step == 0) {
+      $form['text'] = [
```
Comment (lavrenchuk-dev) [inline web/modules/custom/multistep_form/src/Form/MultistepForm.php:47]: I think better will be to move this to access handler
If no steps is available user shouldn't be able to access the form

File: web/modules/custom/multistep_form/src/Form/MultistepForm.php [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+      }
+    }
+    $max_step = count($enabled_config);
+
+    $form['link'] = [
+      '#type' => 'link',
+      '#title' => $this->t('Configure your form here'),
+      '#url' => Url::fromRoute('multistep_form.multistep_form_settings_js'),
+      '#attributes' => ['class' => ['button__configure-form']],
+    ];
```
Comment (lavrenchuk-dev) [inline web/modules/custom/multistep_form/src/Form/MultistepForm.php:44]: Are users without access to the settings will be able to see that?

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]
Overall looks okay, but didn't review JS properly, at first glance it seems like some parts in that JS are not usefull

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**